plugins/katello/developers.md

```diff
@@ -45,53 +42,56 @@ There are a few important states that issues can be in that are worth knowing ab
 
 #### General Flow for New or Needs More Information
 
-The general flow is to open an issue and the runner to exam the issue, typically reading the general idea aloud. This allows for any developers with knowledge to chime in with any relevant info. The runner attempts to answers the following questions:
+The general flow is to open an issue and the runner to examine the issue, typically reading the issue number and general idea aloud. This allows for any developers with knowledge to chime in with any relevant info. The runner attempts to answer the following questions:
 
     * Does this issue need more information from the user?
-        * If yes, leave a comment asking for that information
+        * If yes, leave a comment asking for that information, set the 'Needs More Information' status
     * Does this issue need a developer to investigate it further to properly triage it?
         * If yes, set status to 'Needs Design'
-    * What release should the issue be targeted for? (e.g. next z-stream, current release, next release, backlog)
+    * Which release should the issue be targeted for? (e.g. next z-stream, current release, next release, or no release (implicit backlog))
     * Does any developer want to own it?
         * If yes, assign it
+    * Does the issue have downstream information?
+        * If yes, edit the description to remove the downstream bits
     * Issue category?
+    * Issue priority?
+    * Issue difficulty?
 
 #### General Flow for Ready for Testing
 
   * Will this PR get completed by the next release?
-  * What release should the issue be targeted for? (e.g. next z-stream, current release, next release, backlog)
+  * What release should the issue be targeted for? (e.g. next z-stream, current release, next release)
 
 #### General Flow for Closed
 
-  * Should this issue be targeted at the next z-stream? Current RC phase? Or just throw it in the "next" release (aka develop or master)
+  * What release should the issue be targeted for? (e.g. next z-stream, current release, next release)
 
 ### Notes
 
 A few notes about some automatic operations that are performed by the prprocessor that help facilitate this workflow.
 
-  * All issues are set to an empty release, which again represents "untriaged"
-  * Any issues on the backlog that a PR is opened for has its release set to 'empty' to ensure that it flows through the triage process
-  * Developers that believe an issue should be considered for a release, should set the release to 'empty' and show up to triage to argue their case
+  * All issues are set to an empty release and untriaged
```

Comment:
Did the code get changed to account for this? I would also emphasize that "triaged" is a field vs. the concept of triage

Reply:
`All issues are set to an empty release and untriaged` when? May want to specify